cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+ steps:
+ - name: maven:3-jdk-11
+   entrypoint: mvn
+   args: ['test']
+   dir: 'capstone/'
+ - name: maven:3-jdk-11
+   entrypoint: mvn
+   args: ['package', 'appengine:deploy']
+   dir: 'capstone'
```

Comment:
Let's avoid moving everything, if possible. This is going to make is difficult to merge code Cade/Vincent have on other branches, and this move shouldn't be necessary.